src/components/reusable/daterangepicker/daterangepicker.ts

```diff
@@ -300,6 +299,13 @@ export class DateRangePicker extends LitElement {
   }
 
   private validateStartEndDate(): void {
+    // Save combine values to form data
+    const formData = new FormData();
+    const dateName = this.name ?? 'date-range';
+    const combineVal = `${this.startDate} - ${this.endDate}`;
+    formData.append(dateName, combineVal);
+    this.internals.setFormValue(formData);
```

Comment:
I don't think you need to do both of these things. `setFormValue()` is probably enough by itself.

Reply:
oh yes my bad. Done ..!!